release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -612,37 +612,50 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 
 [id="ocp-4-14-NRO-image-must-gather"]
-==== Collecting data for the NUMA Resources Operator with the must-gather image of the operator
+==== Collecting data for the NUMA Resources Operator with the must-gather image of the Operator
 
-In this release, the `must-gather` tool is updated to collect the data of the NUMA Resources Operator with the `must-gather` image of the operator.
+In this release, the `must-gather` tool is updated to collect the data of the NUMA Resources Operator with the `must-gather` image of the Operator.
 
 For further information about gathering debugging information for the NUMA Resources Operator, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-about-collecting-nro-data_numa-aware[Collecting NUMA Resources Operator data].
 
 [id="ocp-4-14-additional-power-savings-control"]
 ==== Enabling more control over the C-states for each pod
 
-With this release, you have more control over the C-states for your pods. Now, instead of disabling C-states completely, you can specify a maximum latency in microseconds for C-states. You can configure this option in the `cpu-c-states.crio.io` annotation, which helps to optimize power savings in high-priority applications by enabling some of the shallower C-states instead of disabling them completely.
+With this release, you have more control over the C-states for your pods. Now, instead of disabling C-states completely, you can specify a maximum latency in microseconds for C-states. You can configure this option in the `cpu-c-states.crio.io` annotation. This helps to optimize power savings in high-priority applications by enabling some of the shallower C-states instead of disabling them completely.
 
-For further information about enabling more control over the C-states, see xref:../scalability_and_performance/cnf-low-latency-tuning.html#node-tuning-operator-pod-power-saving-config_cnf-master[Optional: Power saving configurations].
+For further information about controlling pod C-states, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#node-tuning-operator-pod-power-saving-config_cnf-master[Optional: Power saving configurations].
 
 [id="ocp-4-14-nw-ipv6-spoke-cluster-support"]
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
 With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.
 
+[id="ocp-custom-crs-with-pgt-ztp"]
+==== Using custom CRs with PolicyGenTemplate CRs in the {ztp-first} pipeline
```

Comment:
I was going to recommend spelling out CRs, but I think that sounds and reads awkward. Not sure of a workaround or rewrite that could help. Maybe `Using customizable custom resources with ....`. That might change the meaning too much.